core/src/main/java/org/apache/iceberg/actions/RewriteFileGroup.java

```diff
@@ -59,6 +62,12 @@ public Set<DataFile> rewrittenFiles() {
         .collect(Collectors.toCollection(DataFileSet::create));
   }
 
+  public Set<DeleteFile> rewritableDeletes() {
```

Comment:
nit: is `orphanedDeleteVectors` a more informative name, since it only counts DVs? Equality delete files aren't included.